common/changes/office-ui-fabric-react/fix-button-that-has-inner-html_2018-08-03-18-49.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Button should not have any html tag as children. In IE the inner html, would not render. ",
```

Comment:
This comment here needs to be adjusted. You're modifying pickers, not button. And even with the Pickers: prefix, what button are you talking about?